frontend/src/pages/AccountSend.vue

```diff
@@ -365,6 +367,12 @@ function useSendForm() {
         advancedAcknowledged.value = false;
       }
 
+      // Switch off the sendMax flag and clear value if we change tokens.
+      if (tokenValue != (_prevTokenValue || prevNativeTokenValue) && sendMax.value) {
```

Comment:
Let's compare the address field specifically to be a bit more robust against surrounding code changes, since this compares by reference by we just care about the values. Also in general recommend always using `!==` to avoid JS type casting for you

![image](https://user-images.githubusercontent.com/17163988/218209012-9306b54c-1fea-45fe-99bd-5c02ad3cfbf7.png)

Reply:
Ah yeah, good catch 👍